topics/admin/tutorials/object-store/tutorial.md

```diff
@@ -326,11 +326,11 @@ This tutorial will help you setup the connection between Galaxy and Dropbox, all
 >    @@ -65,6 +67,12 @@ galaxy_config_templates:
 >       - src: templates/galaxy/config/job_conf.xml.j2
 >         dest: "{{ galaxy_config.galaxy.job_config_file }}"
->     
->    +galaxy_config_files:
->    +  - src: files/galaxy/config/user_preferences_extra_conf.yml
+>
+>    +galaxy_config_templates:
```

Comment:
We have 2 `galaxy_config_templates` sections here, which is probably bad.

Reply:
that one is outside the GAT system. I've reverted it to the previous setup.